Sources/SKCore/ToolchainRegistry.swift

```diff
@@ -108,15 +114,17 @@ extension ToolchainRegistry {
           _default = nil
           return
         }
-        precondition(toolchainIdentifiers[toolchain.identifier] === toolchain,
-          "default toolchain must be registered first")
+        precondition(_toolchains.contains(toolchain), "default toolchain must be registered first")
         _default = toolchain
       }
     }
   }
 
+  /// The default Xcode toolchain identifier.
+  public static let xcodeDefaultToolchainIdentifier: String = "com.apple.dt.toolchain.XcodeDefault"
+
   /// The standard default toolchain identifier on Darwin.
-  public static let darwinDefaultToolchainIdentifier: String = "com.apple.dt.toolchain.XcodeDefault"
+  public static let darwinDefaultToolchainIdentifier: String = xcodeDefaultToolchainIdentifier
```

Comment:
When would you expect these to differ?

Reply:
Good point, I guess in practice we don't expect this to change; reverted